.github/workflows/update-changelog-and-api-version.yml

```diff
@@ -2,14 +2,12 @@ name: update changelog and api version
 
 on:
   push:
-    branches: 
-      - main
-      - 'release/**'
+    branches: [ main, 'sprint/**', 'release/**' ]
+    paths-ignore: ['docs/**', 'Tests/**']
 
   pull_request:
-    branches: 
-      - main
-      - 'release/**'
+    branches: [ main, 'sprint/**', 'release/**' ]
```

Comment:
Pls remove sprint branches for now. We don't want to enforce changelog on sprint branches.